### PR TITLE
Renames everything to besu, still need to wait on changes on pantheon…

### PR DESCRIPTION
**Pull request by josh-richardson**
This PR renames all variables containing "Pantheon" to "Besu".

### CODE REVIEW

File: example/erc20.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ web3.eea
     // contract.methods.transfer(["to", "value"]).send(??)
 
     // already 0x prefixed
+    // eslint-disable-next-line no-underscore-dangle
```
[the remark this answers is not in the document]
Comment (josh-richardson) [reply example/erc20.js]: This is related to eslint not liking underscores which start variable names, as they're usually used to denote private variables.

File: README.md [hunk 2]
```diff
@@ -77,4 +77,4 @@ ok 16 should be equal
 # pass  16
 
 # ok
-```
\ No newline at end of file
+```
```
[the remark this answers is not in the document]
Comment (josh-richardson) [reply README.md]: Unsure. Potentially a line ending?

**Comment by MadelineMurray** [comment]
When I try and run storeValueFromNode1.js, I'm getting the following error: 
```
Madelines-MBP:multiNodeExample madelinemurray$ node storeValueFromNode1.js 
/Users/madelinemurray/eeajs/example/multiNodeExample/storeValueFromNode1.js:13
  const functionAbi = contract._jsonInterface.find(e => {
                                              ^

TypeError: Cannot read property 'find' of undefined
    at storeValueFromNode1 (/Users/madelinemurray/eeajs/example/multiNodeExample/storeValueFromNode1.js:13:47)
    at Object.<anonymous> (/Users/madelinemurray/eeajs/example/multiNodeExample/storeValueFromNode1.js:119:3)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

`deployContract.js` is fine

**Comment by MadelineMurray** [comment]
That issue was resolved. npm test runs.

**Pull request merged**